README.md

```diff
@@ -23,6 +23,8 @@ Note that brevity is not a primary goal. Code should be made more concise only i
 
 ## Swift Package Manager command plugin
 
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fairbnb%2Fswift%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/airbnb/swift) [![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fairbnb%2Fswift%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/airbnb/swift)
+
```

Comment:
Should we put this at the top of this README? That's what I've usually seen in other repos.

Reply:
Sounds good, that works too

Reply:
https://github.com/airbnb/swift/pull/178/commits/c25074138a38e8f6228f701ca04665ff9fd38a14